ddcz/templates/users/awaiting-registrations.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'public.html' %}
+{% load datetime %}
+{% block content %}
+
+<h1 class="page-heading">Dračí líheň</h1>
+<p>
```

Comment:
Missing text?

Reply:
Yup, I was thinking about some nice text how the dragons are hatching, but I haven't figured anything out. So I thought someone could find some good words.